### DISCUSSION TITLE
Using an object store behind MooseFS instead of local fs (RFE?)

### DISCUSSION THREAD

**Discussion by volcimaster** [moosefs/moosefs#496] [Ideas]
Is it possible to use an Object Store (like S3) for the underlying filesystem MooseFS abstracts instead of a local filesystem like ext4?

If not, is it on the roadmap?

**Comment by eleaner** [comment]
out of curiosity, what would be the purpose of such a setup?

**Comment by chogata** [comment]
It's not possible currently directly from MooseFS. It's a feature we have discussed, but there is no firm decision whether it will be implemented or not.

**Comment by footlooseboss** [comment]
[JuiceFS](https://juicefs.com/en/blog/engineering/distributed-filesystem-comparison) might be of interest.
